src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param number numero a ser evaluado.
+     * @return cadena resultado.
+     */
+    public static String singleFizzBuzz(int number) {
+        if (number % MCM == 0) {
+            return "FizzBuzz";
+        } else if (number % THREE == 0) {
+            return "Fizz";
+        } else if (number % FIVE == 0) {
+            return "Buzz";
+        } else {
+            String fbuzz = Integer.toString(number);
+            CharSequence three = "3";
+            CharSequence five = "5";
+            if (fbuzz.contains(three)) {
+                return "Fizz";
+            } else if (fbuzz.contains(five)) {
+                return "Buzz";
+            } else {
+                return fbuzz;
+            }
+        }
```

Comment:
This method may be refactored with OR conditionals.
please take a look at line 30 and 38
also 32 and 40
for both cases is returning the same result.